cms/private_media/static/private_media/js/document-modal-overrides.js

```diff
@@ -0,0 +1,12 @@
+// eslint-disable-next-line
+class DocumentChooserModal extends window.DocumentChooserModal {
+  getURLParams(opts) {
+    const urlParams = super.getURLParams(opts);
+    // eslint-disable-next-line
```

Comment:
Hey @ababic just looking at the js changes here as I was curious. I think these eslint disables might have been needed in the alpha repo due to a `prefer-destructuring` rule in the torchbox eslint setup, but in this repo we're inheriting the wagtail rules so you may find you don't need them. I can help resolve the issue if there is still an error when you remove them.

Reply:
Thanks @helenb. I don't remember the error exactly, but was more to do with referencing window, which isn't defined within the script.

Reply:
Ah you can avoid that by declaring it as a global - see https://eslint.org/docs/latest/rules/no-undef. But again I am not sure this applies in this build so you may just be able to remove the comments completely.